api/v1/controllers/AuthController.js

```diff
@@ -1,8 +1,18 @@
+const MINIMUM_PASSWORD_LENGTH = 8;  // Default minimum length for password
```

Comment:
Do you think this comment is redundant? I figure that the name of the constant is pretty clear